common/corerequestcontext.go

```diff
@@ -5,26 +5,20 @@ import (
 	"net/http"
 
 	"github.com/anz-bank/sysl-go/common/internal"
-
 	"github.com/sirupsen/logrus"
-)
 
-func GetLogEntryFromContext(ctx context.Context) *logrus.Entry {
```

Comment:
I'd like these methods to be deprecated rather than removed and function as they did before. If consumers use a `logrus.Logger` then they should still be able to pull it out of the context.

When deprecating, add clear documentation about what should be done instead (ie. use the `pkg` logger).